packages/gasket-plugin-middleware/README.md

```diff
@@ -69,33 +86,41 @@ export default makeGasket({
 
 ## Middleware paths
 
-The `gasket.js` can contain a `middleware` property, which is an array of
-objects that map plugins to route or path patterns, allowing apps to tune which
-middleware are triggered for which requests.
+You can define middleware paths in your `gasket.js` file using the `middleware`
+property. This property is an array of objects that map plugins to specific
+route or path patterns, allowing you to control which middleware is triggered
+for specific requests.
 
 ```js
+// gasket.js
+
+export default makeGasket({
+  ...
   middleware: [
     {
-      plugin:'gasket-plugin-example', // Name of the Gasket plugin
+      plugin: 'gasket-plugin-example', // Name of the Gasket plugin
       paths: ['/api']
     },
     {
-      plugin:'@some/gasket-plugin-example',
+      plugin: '@some/gasket-plugin-example',
       paths: [/\/default/]
     },
     {
       plugin: '@another/gasket-plugin-example',
       paths: ['/proxy', /\/home/]
     }
   ]
+  ...
+});
+
 ```
 
 ## Lifecycles
 
-### middleware
+### `middleware`
```

Comment:
Stick to no inline code in headings per our [markdown guidelines](https://github.com/godaddy/gasket/blob/v7/CONTRIBUTING.md#markdown-documentation)
```suggestion
### middleware
```